app/controllers/api/v1/reference/locations_controller.rb

```diff
@@ -24,7 +24,7 @@ def render_location(location, status)
           render json: location, status: status
         end
 
-        PERMITTED_FILTER_PARAMS = %i[location_type nomis_agency_id].freeze
+        PERMITTED_FILTER_PARAMS = [:location_type, :nomis_agency_id, supplier_ids: []].freeze
```

Comment:
Maybe this should be singular: `supplier_id`. I don't think we'll ever have a case where we need to filter on multiple suppliers.

A user should never work for both suppliers, in theory.

Reply:
`supplier_id` is something that doesn't exist on that model, as the relationship has been set as a many to many. That would require me intercepting the filter and rewrite the parameter in `supplier_ids`. Can do it, but extremely sloppy - eventually this is not a many to many.